cves/CVE-2016-7905.yml

```
@@ -276,8 +297,12 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: true
```

```suggestion
  answer: false
```
If there is no proof of something we can not assume it is true by default. Most computer systems use the same operating system therefore using the same file formats.